docs/micro-lc/authentication.md

```diff
@@ -117,11 +118,11 @@ that can be defined using the [`aclExpression`](core_configuration.md#aclexpress
 To evaluate, in the `aclExpression` is injected the `groups` object, take from the current user profiles (i.e. `groups.admin && groups.ceo`) 
 :::
 
-The match between the `aclExpression` and the current user profiles is made by **be-config**,
-which must receive them in a header whose name can be customized in **be-config** instance thanks to the environment variable `GROUPS_HEADER_KEY`.
+The match between the `aclExpression` and the current user profiles is made by **be-config**.
+**Be-config** must receive the user profile in a header. Thanks to the environment variable `GROUPS_HEADER_KEY`, you can customize the names of the user profiles in **be-config** instance.
```

Comment:
I know that this is the start of a sentence, but its name is `be-config` lowercase.

Reply:
Ok you're right